hw2/01-flexbox.md

```diff
@@ -2,6 +2,25 @@
 
 1. justify-content: flex-end;
 2. justify-content: center;
-3.
-4.
-5. ...
+3. justify-content: space-around;
+4. justify-content: space-between
+5. align-items: flex-end
+6. justify-content: center; align-items: center
+7. justify-content: space-around;align-items: flex-end
+8. flex-direction: row-reverse;
+9. flex-direction: column;
+10. flex-direction: row-reverse;justify-content: flex-end;
+11. flex-direction: column;justify-content: flex-end;
+12. flex-direction: column-reverse;justify-content: space-between;
+13. flex-direction: row-reverse;justify-content: center; align-items: flex-end
+14. order: 1;
+15. order:-3
+16. align-self: flex-end;
+17. align-self: flex-end; order: 1;
+18. order:1;align-self :flex-end;
+19. flex-wrap: wrap;
+20. flex-wrap: wrap;flex-direction: column
+21. align-content: flex-start;
+22. align-content: flex-end;
+23. flex-direction: column-reverse;align-content: center;
+24. can't solve
```

Comment:
Try using reverse for flex-flow and justify the content to the center and align the spaces.

Reply:
Thanks it helps updated the code